lib/codecs/tests/native.rs

```diff
@@ -22,6 +22,11 @@ fn pre_v34_fixtures_match() {
     fixtures_match("pre-v34");
 }
 
+#[test]
+fn pre_v41_fixtures_match() {
+    fixtures_match("pre-v41");
+}
```

Comment:
I think we also need tests of `roundtrip_fixtures` and `decoding_matches`.

Reply:
It looks like the `decoding_matches` tests are broken due to https://github.com/vectordotdev/vector/issues/18570. Will add the `ignore` flag for the `pre-v41` test as well